docs/hello-sample.md

```diff
@@ -1,150 +1,300 @@
-# Running the 'Hello' Sample
+# Running the 'Hello' Samples
 
-Here are simple instructions for running the 'Hello' sample project. We included scripts that make it easy to build, run, and deploy this application. Also, this sample is a great resource for understanding how Durable Functions and Netherite work, and an excellent starting point for creating your own projects.
+In order to speed up your first experience with Netherite we provide some samples that serve this goal. They are also a great resource for understanding how Durable Functions and Netherite work. They are also excellent starting point for creating your own projects.
```

Comment:
Avoid starting sentence with "They are also" twice in a row.

Reply:
Fair point.